### ISSUE TITLE
Delete row with a prepared sql

### ISSUE THREAD

**Issue by asanchezgomez** [jayralencar/sqlite-cipher.js#2]
Hi!
I've tried to **delete** a row in a database with a prepared sql, but I'm not able to do it and I have not found any documentation about delete database function with your sqlite-cipher package. Here you can find the code:

```
var sqlite = require('sqlite-cipher');

  sqlite.connect('myEncrypedFile.db','MyPassword');
  sqlite.run("CREATE TABLE COMPANYS(ID  INTEGER PRIMARY KEY AUTOINCREMENT, NAME TEXT NOT NULL);");
  sqlite.insert("COMPANYS",{NAME:"My COMPANY"});
  
  var rows = sqlite.run("SELECT * FROM COMPANYS;");
	for(var i = 0; i < rows.length; i++)
		console.log(rows[i].NAME);
	
	//Delete row
	var name = "My COMPANY";

	//First attempt --> it does not work
	sqlite.run("DELETE FROM COMPANYS WHERE NAME = '?'",[name]);
	rows = sqlite.run("SELECT * FROM COMPANYS;");
	console.log("First attempt:");
	for(var i = 0; i < rows.length; i++)
		console.log(rows[i].NAME);

	//Second attempt --> it does not work
	sqlite.run("DELETE FROM COMPANYS WHERE NAME = ?",[name]);
	console.log("Second attempt:");
	rows = sqlite.run("SELECT * FROM COMPANYS;");
	for(var i = 0; i < rows.length; i++)
		console.log(rows[i].NAME);

	//Third attept --> it works but it is not a prepared sql
	var sql = "DELETE FROM COMPANYS WHERE NAME = '"+name+"'";
	sqlite.run(sql);
	console.log("Third attempt:");
	rows = sqlite.run("SELECT * FROM COMPANYS;");
	for(var i = 0; i < rows.length; i++)
		console.log(rows[i].NAME);
```
Thanks in advance!

**Comment by jayralencar** [comment]
Hello @asanchezgomez , thanks for report.This problem is solved, install version 0.2.0.

**Comment by asanchezgomez** [comment]
Thanks for your comment! It works nice with the first approach:

```
var sqlite = require('sqlite-cipher');

  sqlite.connect('myEncrypedFile.db','MyPassword');
  sqlite.run("CREATE TABLE COMPANYS(ID  INTEGER PRIMARY KEY AUTOINCREMENT, NAME TEXT NOT NULL);");
  sqlite.insert("COMPANYS",{NAME:"My COMPANY"});

  var rows = sqlite.run("SELECT * FROM COMPANYS;");
    for(var i = 0; i < rows.length; i++)
        console.log(rows[i].NAME);

    //Delete row
    var name = "My COMPANY";

    //First attempt --> it works with 0.2.0 version
    sqlite.run("DELETE FROM COMPANYS WHERE NAME = '?'",[name]);
    rows = sqlite.run("SELECT * FROM COMPANYS;");
    console.log("First attempt:");
    for(var i = 0; i < rows.length; i++)
        console.log(rows[i].NAME);


```
